modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -0,0 +1,222 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-controller-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-autoscaling-ingress-controller_{context}"]
+= Autoscaling an Ingress Controller
+
+Automatically scale an Ingress Controller to dynamically meet routing performance or availablility requirements such as the requirement to increase throughput. The following procedure provides an example for scaling up the default `IngressController`.
+
+.Prequisites
+. You installed the OpenShift CLI (`oc`).
+. You have access to the cluster as a user with the `cluster-admin` role.
```

Comment:
```suggestion
. You have access to an {product-title} cluster as a user with the `cluster-admin` role.
```
You can also swap out `role ` with `permissions` but that is totally up to you. We use `permissions` a lot in the web console sections.